### ISSUE TITLE
Cannot install with yarn because of "link" dependency

### ISSUE THREAD

**Issue by webmaster128** [jehon/karma-jasmine-es6#1]
I just tried to install this in yarn project:

```
$ yarn add karma-jasmine-es6 --dev
yarn add v1.6.0
[1/4] Resolving packages...
[2/4] Fetching packages...
info fsevents@1.2.4: The platform "linux" is incompatible with this module.
info "fsevents@1.2.4" is an optional dependency and failed compatibility check. Excluding it from installation.
error link@0.1.5: The engine "node" is incompatible with this module. Expected version "> 0.4.x < 0.9.0".
error Found incompatible module
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
```

In contrast to npm, yarn sees the engines field in packages.json as a strict requirement.

The dependency "link" [sets](https://github.com/calyptus/link.js/blob/master/package.json) `engines` to ` { "node": "> 0.4.x < 0.9.0" }`, which is very much outdated.

Any idea how to install this with yarn? Do we really need the dependency "link"?

**Comment by jehon** [comment]
I will look into this tonight...

Thanks for your feedback
Jean

**Comment by jehon** [comment]
It is fixed in the version 0.0.3.
The fact was that the "link" dependancy was not usefull in non-dev usage.

Enjoy!
